android/sdl_android/src/main/java/com/smartdevicelink/managers/video/VideoStreamManager.java

```diff
@@ -377,18 +488,37 @@ public void stopStreaming() {
         stateMachine.transitionToState(StreamingStateMachine.STOPPED);
     }
 
+    /**
+     * Stops streaming from the remote display. To restart, call
+     *
+     * @see #resumeStreaming()
+     */
+    public void stopStreaming(boolean withPendingRestart) {
```

Comment:
Hello @kostyaBoss I dont this that this should be made public as this should only ever be called with `TRUE` from within the library and for any cases where the developer would be calling this method with `FALSE` I believe it would be better to use the existing `stopStreaming` method.
I propose removing the deprecated annotation from the existing `stopStreaming` method and making this method private